oval_graph/oval_tree/oval_node.py

```diff
@@ -1,4 +1,5 @@
-import oval_graph.evaluate
+from .evaluate import (is_notapp_result, oval_operator_and, oval_operator_one,
```

Comment:
I would use `from . import evaluate` - it's quite easy to write, and it also puts only one symbol to the global namespace.